### ISSUE TITLE
TASTy-MiMa integration

### ISSUE THREAD

**Issue by grzegorz-bielski** [scalacenter/sbt-version-policy#204]
Hello!

I noticed there already exists [sbt-tasty-mima](https://github.com/scalacenter/sbt-tasty-mima) plugin for checking compatibilities between TASTy files.

Are there any plans to integrate it for Scala 3 with [sbt-version-policy](https://github.com/scalacenter/sbt-version-policy) along with existing MiMa checks?

**Comment by julienrf** [comment]
I agree it would be good to include TASTy compatibility checks in sbt-version-policy. Nobody is actively working on sbt-version-policy, as far as I know, but such a contribution would be welcome!
